engines/grim/gfx_opengl.cpp

```diff
@@ -1957,33 +1957,12 @@ void GfxOpenGL::drawRectangle(const PrimitiveObject *primitive) {
 		glVertex2f(x1, y2 + 1);
 		glEnd();
 	} else {
-		glBegin(GL_QUADS);
-
-		// top line
+		glLineWidth(_scaleW);
```

Comment:
Good question. I took this call from GfxOpenGL::drawLine just below. Maybe I should add a comment about this approximation ? And should I withdraw the opengl part of my change, as it does not change the rendering, but merely tries to avoid increasing the diff between OpenGL and TinyGL ?